### PR TITLE
Added css styles to views

### PR DESCRIPTION
**Pull request by astro6703**
player index
player viewall
home
match bestofn
match create
match index
match viewall
player create

### CODE REVIEW

File: SeasonStats/Views/Match/ViewAll.cshtml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </tr>
+    </thead>
+    <tbody>
+            @for (int i = 1; i <= ViewBag.Matches.Count; i++)
+            {
+                <tr>
+                    <td class="number">@(i)</td>
+                    <td class="players">@(ViewBag.Matches[i - 1].Player1.Name)</td>
+                    <td class="scores">
+                    <div id="player1score">@(ViewBag.Matches[i - 1].Player1Score)</div>
```
Comment (kapral) [inline SeasonStats/Views/Match/ViewAll.cshtml]: you can't use `id` here as it's in a loop, multiple elements can't have same id on a page - that's invalid html

File: SeasonStats/wwwroot/css/Home/Index.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.mainmenuitems {
+    display: block;
+    text-align: center;
+    font-size: 20px;
+    color: black;
+    text-decoration: none;
+    background-color: lightgray;
+    margin: 0.5% auto 0.5% auto;
```
Comment (kapral) [inline SeasonStats/wwwroot/css/Home/Index.css]: does this relative vertical margin work?

File: SeasonStats/wwwroot/css/Home/Index.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    width: 30%;
+    line-height: 200%;
```
Comment (kapral) [inline SeasonStats/wwwroot/css/Home/Index.css]: never seen usage line height in percents, is it same as `2`?

File: SeasonStats/wwwroot/css/Player/Index.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
+﻿#parent
+{
+    font-size: 0px;
+}
+
+#mainmenucontainer {
+    width: 100%;
+}
+
+.mainmenuitems {
```
Comment (kapral) [inline SeasonStats/wwwroot/css/Player/Index.css]: some styles are copied, could be shared

File: SeasonStats/wwwroot/css/Home/Index.css (hunk 2 above)
Comment (astro6703) [reply SeasonStats/wwwroot/css/Home/Index.css]: yes it does

**Pull request merged**